community/releases/release-manager-runbook/index.md

```diff
@@ -295,9 +294,8 @@ channel to let them know that a new patch is available.
         - BOM_VERSION: This is the BOM to associate the Spin CLI release with. It is
         the latest Spinnaker release number, 1.xx.x.
 
-1. Publish a Sponnet minor version. Run the [publish.sh](https://github.com/spinnaker/sponnet/blob/master/publish.sh)
-script while passing in the version number of the new release.
-Example: VERSION="1.17.2" ./publish.sh
+1. Make a Sponnet [github release](https://github.com/spinnaker/sponnet/releases/new). Give it the same version as the newly released Spinnaker, with the tag prefixed with "v" (i.e. v${RELEASE}).
```

Comment:
```suggestion
1. Make a Sponnet [GitHub release](https://github.com/spinnaker/sponnet/releases/new). Give it the same version as the newly released Spinnaker, with the tag prefixed with "v" (e.g., v1.22.0).
```

Reply:
Prefer "for example," rather than "e.g.," or "i.e.".